.github/workflows/tests.yml

```diff
@@ -0,0 +1,54 @@
+name: "Run tests (pytest)"
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+        matrix:
+            os: [ubuntu-latest, macos-latest]
+            python-version: ["3.8", "3.10"]
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
```

Comment:
Is there any difference is we use `version 3` or `version 1` instead of `version 2`?

Reply:
I'll have to pass on that one..